rebuild.sh

```diff
@@ -86,6 +86,15 @@ fi
 
 echo -e "\033[1m$(pwd)\033[0m"
 
+containerEngine() {
```

Comment:
given `${container_engine}` is used in other location, this function that is used in only one location is a duplicate: I feel it's better to drop it in favor of `${container_engine}` for every caller, isn't it?

Reply:
Yes, definitely! Thank you for your comment. I removed this method which is of no use. I still have an error while executing the command, so I'm keeping this PR as a draft until I find a way to make it work :)